core/modules/hubot/message.js

```diff
@@ -0,0 +1,14 @@
+﻿var Message = function (event, prefix) {
+    this.event = event;
+    this.prefix = prefix;
+
+    this.room = event.thread_id;
+    this.text = event.body;
+    this.user = {
+        name: event.sender_name,
+        id: event.sender_id,
+        email_address: 'kassy@foobar.com'
+    };
+};
+
+module.exports = Message;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/theprojectshop/Kassy/file/2530104136/issues/source?bid=3282987&fileBranchId=3282997#l14)